src/renderer/styles/base.css

```
@@ -0,0 +1,198 @@
+  --space-6: 14px;
+  --space-7: 16px;
+  --space-8: 18px;
+  --space-9: 20px;
+  --space-10: 24px;
+  --space-11: 28px;
+  --space-12: 32px;
+  --route-rail-width: 236px;
+  --screen-min-height: calc(100vh - 116px);
+  --button-min-height: 42px;
+  --transition-fast: 120ms ease;
+  color-scheme: dark;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)